src/Analyzer.Core/Rules/BuiltInRules.json

```diff
@@ -86,6 +86,55 @@
       "equals": true
     }
   },
+  {
+    "name": "Cache_Redis_SSL_Only",
+    "description": "Only secure connections to your Azure Cache for Redis should be enabled",
+    "recommendation": "Enable connections via SSL only to Redis Cache",
+    "helpUri": "https://github.com/Azure/template-analyzer/docs/built-in-bpa-rules.md/#only-secure-connections-to-your-azure-cache-for-redis-should-be-enabled",
+    "evaluation": {
+      "resourceType": "Microsoft.Cache/redis",
+      "path": "properties.enableNonSslPort",
+      "notEquals": true
+    }
+  },
+  {
+    "name": "Compute_VirtualMachine_ShouldNotBePreview",
+    "description": "Ensure all virtual machines are not using preview images",
+    "recommendation": "Use a non-preview image for virtual machines",
+    "helpUri": "https://github.com/Azure/template-analyzer/docs/built-in-bpa-rules.md/#ensure-all-virtual-machines-are-not-using-preview-images",
+    "evaluation": {
+      "resourceType": "Microsoft.Compute/virtualMachines",
+      "anyOf": [
+        {
+          "path": "properties.storageProfile.imageReference",
+          "regex": "^((?!-preview$).)*$"
+        },
+        {
+          "path": "properties.storageProfile.imageReference.version",
+          "regex": "^((?!-preview$).)*$"
+        }
+      ]
+    }
+  },
+  {
+    "name": "Compute_VirtualMachineScaleSets_ShouldNotBePreview",
+    "description": "Ensure all virtual machine scale set instances are not using preview images",
+    "recommendation": "Use a non-preview image for virtual machine scale set instances",
+    "helpUri": "https://github.com/Azure/template-analyzer/docs/built-in-bpa-rules.md/#ensure-all-virtual-machine-scale-set-instances-are-not-using-preview-images",
+    "evaluation": {
+      "resourceType": "Microsoft.Compute/virtualMachineScaleSets",
+      "anyOf": [
```

Comment:
@JohnathonMohr's comment from previous PR

I'm fairly confident the TTK is incorrectly checking this property. But fortunately I don't think there's any harm keeping this format for now.

Reply:
We are still pending confirmation if this rule is 100% correct.